build/flaws.js

```diff
@@ -561,8 +561,8 @@ async function fixFixableFlaws(doc, options, document) {
         if (response && response.statusCode === 404) {
           console.log(chalk.yellow(`Skipping ${flaw.src} (404)`));
           continue;
-        } else if (error.code === "ETIMEDOUT") {
-          console.log(chalk.yellow(`Skipping ${flaw.src} (request timed out)`));
+        } else if (error.code === "ETIMEDOUT" || error.code === "ENOTFOUND") {
+          console.log(chalk.yellow(`Skipping ${flaw.src} (${error.code})`));
```

Comment:
Alternately, this log and continue could just be moved down to the general error